.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: Check PR title and project linting
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  lint:
+    if: "!contains(github.event.pull_request.head.ref, 'renovate')"
```

Comment:
We also have dependabot in civil-ccd-definition. You might want to exclude that one too.

Reply:
added now